### ISSUE TITLE
Broken SyntaxHighlighter

### ISSUE THREAD

**Issue by rchovan** [thorsten/phpMyFAQ#936]
Hi,
after upgrade from 2.8.15 to 2.8.17 is SyntaxHighlighter broken. 
This is how I see it when I edit faq, which is ok,
![Edit mode] (http://i.imgur.com/oihl5Br.png)

and this is visible on frontpage:
![frontpage mode] (http://i.imgur.com/tgGbekr.png)

this is html code in editor
```
<p>reštartovať</p>
<pre class="brush: plain;fontsize: 100; first-line: 1; ">psskr79428#relo <strong>ENTER</strong>
Proceed with reload? [confirm] <strong>ENTER</strong>
</pre>
```
Running on nginx 1.6.2, and it was correctly displayed in 2.8.15.

**Comment by rchovan** [comment]
please reopen, 
discussion is here> http://forum.phpmyfaq.de/viewtopic.php?f=3&t=17460

**Comment by thorsten** [comment]
Okay, the view in the TinyMCE editor is wrong.

**Comment by thorsten** [comment]
I would like to close this issue because we'll remove the Syntax Highlighter in 2.9 because the TinyMCE plugin is broken (and some stuff more...)

**Comment by rchovan** [comment]
So, what should I use next ? Do you plane any replacement ?
Dňa 27.5.2015 21:10 používateľ "Thorsten Rinne" <notifications@github.com>
napísal:

**Comment by thorsten** [comment]
Yes, we'll add a new and better syntax highlighter, maybe the same we use on phpmyfaq.de: https://highlightjs.org/

**Comment by rchovan** [comment]
Ok, close this issue.
Dňa 27.5.2015 21:45 používateľ "Thorsten Rinne" <notifications@github.com>
napísal:

**Comment by thorsten** [comment]
Thanks Mate!

**Comment by rchovan** [comment]
Hi, 
I solved this issue. I have discovered, when I use empty brush style "brush: ;", then is FAQ displayed correctly in editor and on frontpage
`````
<pre class="brush: ;fontsize: 100; first-line: 1; ">bla
<strong>bla</strong>
<em>bla</em></pre>
`````

So I modified dialog.htm 
from this:
`````
<select name="syntaxhl_language" id="syntaxhl_language">
<------><------><------><option value="bash">Bash(Shell)</option>
`````
to this
`````
<select name="syntaxhl_language" id="syntaxhl_language">
<------><------><------><option value="">None</option>
<------><------><------><option value="bash">Bash(Shell)</option>
``````
And when I use brush as None, then everything is ok

Can you apply this patch to future releases ? Thank You

**Comment by thorsten** [comment]
Fixed for 2.8.25, thanks a lot!
